common/terms.html

```diff
@@ -1,5 +1,5 @@
 <section><h3>Terms imported from Other Specifications</h3>
-<p>Terms imported from [[[ECMASCRIPT]]] [[ECMASCRIPT]], [[[RFC8259]]] [[RFC8259]], [[[INFRA]]] [[INFRA]], and [[[WEBIDL]]] [[WEBIDL]]</p>
+<p>Terms imported from [[[ECMASCRIPT]]] [[ECMASCRIPT]], [[[RFC8259]]] [[RFC8259]], [[[INFRA]]] [[INFRA]], [[XPATH-FUNCTIONS]], and [[[WEBIDL]]] [[WEBIDL]]</p>
```

Comment:
It seems odd that there's a `[[[ ]]]` `[[ ]]` for everything but `[[XPATH-FUNCTIONS]]`. Should it have a preceding `[[[XPATH-FUNCTIONS]]]`?